openslides/core/static/js/core/site.js

```diff
@@ -351,6 +351,64 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+.provider('MultiselectFilterProvider', [
+    function () {
+        this.$get = function () {
+            var instance = function () {
+                var filters = {};
+                var getFilters = function () { return filters; };
+                var itemId = {
+                    group: function (obj) {return obj.groups_id;},
+                    state: function (obj) {return obj.state_id;},
+                    category: function (obj) {return obj.category_id;},
+                    motionBlock: function (obj) {return obj.motion_block_id;},
+                    tag: function (obj) {return obj.tags_id;}
```

Comment:
mmh, is this good to use in core such specific keys from our motion and users app?